### DISCUSSION TITLE
License of software must NOT be agreed?

### DISCUSSION THREAD

**Discussion by Speed7811** [microsoft/winget-pkgs#147127] [Q&A] [answered]
Dear all,

I've a small software published in the winget repo. My license url is included in my manifest file. Now when I install my software via "winget install myproduct" I don't get any question if I accept the license or not. I saw that some other products from the msstore has such an restriction. Is this possible with the "winget-store"? If yes what do I need to change in my manifest to get it?

Best regards

Rainer

**Comment by Trenly** [comment, marked as answer]
> Dear all,
> 
> I've a small software published in the winget repo. My license url is included in my manifest file. Now when I install my software via "winget install myproduct" I don't get any question if I accept the license or not. I saw that some other products from the msstore has such an restriction. Is this possible with the "winget-store"? If yes what do I need to change in my manifest to get it?
> 
> Best regards
> 
> Rainer

In order to do this you will need to add the `Agreements` section like this:
```
Agreements:
  AgreementLabel: <Optional label>
  AgreementUrl: <URL To the agreement>
```

This is only required if you want explicit agreement; Otherwise, every install through WinGet does state:
> This application is licensed to you by its owner.

So even in the cases where agreements are not specified, the user is informed that by using the application they are agreeing to the license terms

**Comment by Speed7811** [comment]
Hey,

perfect... thats what I needed. Thank you so much.

Best regards

Rainer
